File: Raven.Client.Java/src/test/java/net/ravendb/tests/bugs/transformresults/AnswerEntity.java
package net.ravendb.tests.bugs.transformresults;

public class AnswerEntity {
  private String id;
  private String userId;
  private Question question;
  private String content;
  public String getId() {
    return id;
  }
  public void setId(String id) {
    this.id = id;
  }
  public String getUserId() {
    return userId;
  }
  public void setUserId(String userId) {
    this.userId = userId;
  }
  public Question getQuestion() {
    return question;
  }
  public void setQuestion(Question question) {
    this.question = question;
  }
  public String getContent() {
    return content;
  }
  public void setContent(String content) {
    this.content = content;
  }

}

File: test/SlowTests/Tests/Bugs/TransformResults/AnswerEntity.cs
namespace SlowTests.Tests.Bugs.TransformResults
{
    public class AnswerEntity
    {
        public string Id { get; set; }
        public string UserId { get; set; }
        public Question Question { get; set; }
        public string Content { get; set; }
    }
    public class AnswerEntity2
    {
        public string Id { get; set; }
        public string UserId { get; set; }
        public Question2 Question { get; set; }
        public string Content { get; set; }
        public AnswerVoteEntity2[] Votes { get; set; }

    }
}
